File: mario_less.py
from cs50 import get_int

n = 0

while(n < 1 or n > 8):
    n = get_int("Height: ")

for i in range(n):
    for j in range(n):
        if (j+i+1) < n:
            print(" ",end="")
        elif (j+i+1>= n):
            print("#", end="")
        if (j+1) == n:
            print("")

File: pset1/mario_less.c
#include <cs50.h>
#include <stdio.h>

void espaco(int n);
void sustenido(int n);

int main(void)
{
    
    int sif = 0, prints, contador, i, j;
    do
    {
        sif = get_int("Height: ");
    }
    while(sif<1 || sif>8);
    
    i=0;
    j=sif-1;
    do
    {
        espaco(sif-i-1);
        sustenido(sif-j);
        j--;
        i++;
    }
    while(i<sif);
}

void espaco(int n)
{
    int i;
    for(i=0;i<n;i++)
    {
        printf(" ");
    }
}

void sustenido(int n)
{
    int i;
    for(i=0;i<n;i++)
    {
        printf("#");
    }
    printf("\n");
}
